Warnings possibly indicating problems

This concatenation of strings is discarded:
https://github.com/ANTsX/ANTs/blob/2d3cefaec09b1652fcc88cdb802b622d32a4cbb5/Examples/Atropos.cxx#L1335-L1344
Maybe a semicolon on the previous line (1334) should be replaced by a plus sign?

Current Visual Studio (17.8.4) reports the following warning:
`C:\Dev\ANTs\Examples\sccan.cxx(2593,8): warning C4858: discarding return value: The 'remove' and 'remove_if' algorithms return the iterator past the last element that should be kept. You need to call container.erase(result, container.end()) afterwards. In C++20, 'std::erase' and 'std::erase_if' are simpler replacements for these two steps.`
That code should be checked, and if it is OK that line should be replaced by:
```cpp
std::ignore = std::remove(args.begin(), args.end(), std::string(""));
```

@cookpa There is just one warning when compiling ANTs normally: `C:\Dev\ANTs\Examples\sccan.cxx(2593,8): warning C4858: discarding return value`. I am not familiar with this code to be able to tell whether we should just ignore the warning.

--- a/Examples/sccan.cxx
+++ b/Examples/sccan.cxx
@@ -2590,7 +2590,7 @@ sccan(std::vector<std::string> args, std::ostream * /*out_stream = nullptr */)
   // which the sccanparser should handle
   args.insert(args.begin(), "sccan");
 
-  std::remove(args.begin(), args.end(), std::string(""));
+  std::ignore = std::remove(args.begin(), args.end(), std::string(""));
   int     argc = args.size();
   char ** argv = new char *[args.size() + 1];
   for (unsigned int i = 0; i < args.size(); ++i)

--- a/SuperBuild/External_ITKv5.cmake
+++ b/SuperBuild/External_ITKv5.cmake
@@ -152,7 +152,7 @@ if(NOT DEFINED ${extProjName}_DIR AND NOT ${USE_SYSTEM_${extProjName}})
 
   ### --- End Project specific additions
   set(${proj}_REPOSITORY ${git_protocol}://github.com/InsightSoftwareConsortium/ITK.git)
-  set(${proj}_GIT_TAG d45127b5a3296b7f1721b57e0ffe8e0071d0c64f)  # 2023-12-27
+  set(${proj}_GIT_TAG 66ab5b14c476d057ed46a4daba05b710681160e6)  # 2024-01-17
   set(ITK_VERSION_ID ITK-5.4) ### NOTE: When updating GIT_TAG, also update ITK_VERSION_ID if ITK version has changed
 
   ExternalProject_Add(${proj}

